gui/src/app/FileEditor/ToolBar.tsx

```diff
@@ -10,12 +23,12 @@ export type ToolbarItem =
       label?: string;
       icon?: any;
       onClick: () => void;
-      color?: string;
+      color?: ColorOptions;
```

Comment:
At this point, probably more semantic to call it something like "Styling" or "Role"

Reply:
Well, it is still a color, just a color palette selection rather than a raw color name

Reply:
Fair enough